compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/WindowInfo.skiko.kt

```diff
@@ -39,6 +39,12 @@ actual interface WindowInfo {
      */
     actual val isWindowFocused: Boolean
 
+    /**
+     * Indicates if window is transparent.
+     */
+    @ExperimentalComposeUiApi
+    val isWindowTransparent: Boolean
```

Comment:
This seems like a rather specialized property.

I share the same concern regarding `Window(transparent = true)`. Ideally, we should use Window(color = Color.White). Or make window transparent by default, and configure it via content's background.